### PR TITLE
add other spaces to admin dashboard

### PR DESCRIPTION
**Pull request by denisdimitrov**
- Adds the rest of the space templates to the `Space dashboard` in the admin.
- ~Removes the `Spaces` and `Run` tabs for templates that are `!== partymap` when managing a space because they make no sense to be shown when for those templates, since they have no rooms or a map.~

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1437

### CODE REVIEW

**Comment by whoaitsaimz** [comment]
Hey @denisdimitrov the first bullet is all that is needed for this PR

What the “edit space” button does for the other template types is beyond the scope of this issue and what you assumed in the second bullet is not what is being asked

**Comment by denisdimitrov** [comment]
@whoaitsaimz I understand, I will revert the changes and only display the spaces for now.

**Comment by ermiaSparkle** [comment]
@denisdimitrov 
Would u please update the headers with these name pic from the issue?
image

File: src/pages/SpacesDashboard/SpacesDashboard.tsx [hunk 1]
```diff
@@ -117,20 +126,31 @@ export const SpacesDashboard: React.FC = () => {
...
                 <div className="SpacesDashboard__welcome-message">
                   Create your first Sparkle space
                 </div>
               </>
             )}
-            {hasVenues && renderedPartyVenues}
+            {hasPartyVenues && renderedPartyVenues}
+          </div>
+
+          {hasOtherVenues && <AdminTitle>Other maps</AdminTitle>}
```
Comment (colinhowe) [inline src/pages/SpacesDashboard/SpacesDashboard.tsx:147]:
```suggestion
          {hasOtherVenues && <AdminTitle>My other spaces</AdminTitle>}
```

File: src/pages/SpacesDashboard/SpacesDashboard.tsx [hunk 2]
```diff
@@ -117,20 +126,31 @@ export const SpacesDashboard: React.FC = () => {
               Settings
             </ButtonNG>
           </AdminTitleBar>
+
+          {hasPartyVenues && <AdminTitle>Party maps</AdminTitle>}
```
Comment (colinhowe) [inline src/pages/SpacesDashboard/SpacesDashboard.tsx:130]:
```suggestion
          {hasPartyVenues && <AdminTitle>My map spaces</AdminTitle>}
```

**Review by colinhowe — APPROVED** [review]
Apart from copy change, this looks great. Make the copy change and then this can be merged without further review.

**Pull request merged**